src/factories/calorimetry/ImagingTopoCluster_factory.h

```diff
@@ -24,8 +24,13 @@ class ImagingTopoCluster_factory
   // ParameterRef<std::vector<double>> m_ldxy{this, "localDistXY", config().localDistXY};
   ParameterRef<std::vector<double>> m_ldep{this, "layerDistEtaPhi", config().layerDistEtaPhi};
   // ParameterRef<std::vector<double>> m_ldxy_adjacent{this, "layerDistXY", config().layerDistXY};
-  ParameterRef<eicrecon::ImagingTopoClusterConfig::ELayerMode> m_laymode{this, "layerMode",
-                                                                         config().layerMode};
+  // ParameterRef<eicrecon::ImagingTopoClusterConfig::ELayerMode> m_laymode{this, "layerMode",
+  //                                                                        config().layerMode};
+  ParameterRef<eicrecon::ImagingTopoClusterConfig::ELayerMode> m_sameLayMode{
+      this, "sameLayerMode", config().sameLayerMode};
+  ParameterRef<eicrecon::ImagingTopoClusterConfig::ELayerMode> m_diffLayMode{
+      this, "diffLayerMode", config().diffLayerMode};
```

Comment:
This is invisible but it's just better to give this the same name.
```suggestion
  ParameterRef<eicrecon::ImagingTopoClusterConfig::ELayerMode> m_sameLayerMode{
      this, "sameLayerMode", config().sameLayerMode};
  ParameterRef<eicrecon::ImagingTopoClusterConfig::ELayerMode> m_diffLayerMode{
      this, "diffLayerMode", config().diffLayerMode};
```